common/src/main/java/io/druid/metadata/DefaultPasswordProvider.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
We haven't done it for all files yet. Your suggestion to use the ASF style came after we had already updated the files. I figured we would update them going forward as we get more contributions, rather than update everything again. If people feel strongly about updating everything we can do that too.